core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -82,6 +85,7 @@ static function (string $v): int {
                 ->end()
                 ->scalarNode('preview_script')
                     ->cannotBeEmpty()
+                    ->info('The default preview script is preview.php. This allows to define a custom script if desired.')
```

Comment:
*Entry point script that bypasses the front end cache for preview features, if necessary. The Contao Managed Edition uses "preview.php" here, but it can be customize for a regular Symfony application.*